5.5/Dockerfile

```diff
@@ -20,10 +20,23 @@ LABEL io.k8s.description="MySQL is a multi-user, multi-threaded SQL database ser
       io.openshift.expose-services="3306:mysql" \
       io.openshift.tags="database,mysql,mysql55"
 
+# Support for atomic run/install/uninstall
+LABEL INSTALL="docker run -t -i --rm --privileged -u 0:0 -v /:/host --net=host \
```

Comment:
@hhorak my concerns with this is:

1) this is not backward compatible and not namespaced, so everytime something changes in the Docker CLI, this will break terribly
2) A lot of repetition (why I need to write `docker run` three times if it is obvious this will run with docker)
3) What will happen if we start supporting rocket or another container provider? Will the INSTALL become `INSTALL.rocket=` ? 
4) Having this labels in one liner saves some time during a build, but parsing this just with eyes is very hard. I mean I completely missed RUN ;-)
5) The OPT1 OPT2... are totally cryptic and I have no idea what they are doing (need to go check atomic docs)
6) These labels will be copied to any other image that will layer on top of this image. If you change this label in the future, the 'upper' images will have to rebuild. There is no way to force rebuild of all images users will publish on Docker Hub, so this might be a huge problem in the future.
7. Why this has to be privileged? We spend a lot of effort to make these images run under arbitrary UID, now we are saying that they should run as root?